main.py

```
@@ -356,6 +364,61 @@ def about_game(matches: List[str]) -> List[str] | None:
+        maybe_title = matches[0]
+        if maybe_title and maybe_title.strip():
+            resolved = resolve_game_interactively(maybe_title)
+            if resolved:
+                g = resolved
+                _last_selected_game = g
+    if not g:
+        return ["No answers"]
+    try:
+        from game_editor import open_edit_ui_with_existing_json  # type: ignore
+        open_edit_ui_with_existing_json(g)
+        return None
```

Ruff (0.14.0)

Consider moving this statement to an `else` block

(TRY300)